pkg/conn/baseconn.go

```diff
@@ -11,61 +11,59 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package baseconn
+package conn
 
 import (
 	"database/sql"
 
-	"go.uber.org/zap"
-
 	tcontext "github.com/pingcap/dm/pkg/context"
 	"github.com/pingcap/dm/pkg/log"
 	"github.com/pingcap/dm/pkg/retry"
 	"github.com/pingcap/dm/pkg/terror"
 	"github.com/pingcap/dm/pkg/utils"
+
+	"go.uber.org/zap"
 )
 
 // BaseConn wraps a connection to DB
+// For Syncer and Loader Unit, they both have different amount of connections due to config
+// Currently we have some types of connections exist
+// Syncer:
```

Comment:
Is `fromDB` should be documented here?

Reply:
`fromDB` wraps `*sql.DB`, maybe after we change it to `BaseConn`, we can document here